Move the buttons and the labels at the bottom of the customer cheque screen into a
single row, saving some screen real-estate.
--- quasar-1.4.7_GPL.orig/gui/cheque_customer.cpp	2004-03-05 10:50:58.000000000 +0200
+++ quasar-1.4.7_GPL/gui/cheque_customer.cpp	2007-03-08 10:35:59.000000000 +0200
@@ -197,8 +197,8 @@
     totalGrid->setColStretch(2, 1);
     totalGrid->addWidget(payAll, 0, 0);
     totalGrid->addWidget(clearAll, 0, 1);
-    totalGrid->addWidget(allocLabel, 1, 3, AlignRight | AlignVCenter);
-    totalGrid->addWidget(_alloc, 1, 4);
+    totalGrid->addWidget(allocLabel, 0, 3, AlignRight | AlignVCenter);
+    totalGrid->addWidget(_alloc, 0, 4);
 
     vbox2->addWidget(_table);
     vbox2->addWidget(totals);
